hafnian/tests/test_samples.py

```diff
@@ -83,7 +83,9 @@ def test_hafnian_sample_states_nonsquare(self):
 
     def test_hafnian_sample_states_nans(self):
         """test exception is raised if not a numpy array"""
-        with pytest.raises(ValueError, match="Covariance matrix must not contain NaNs."):
+        with pytest.raises(
+            ValueError, match="Covariance matrix must not contain NaNs."
+        ):
```

Comment:
@nquesada - it looks like your Black settings might be off, as these formatting changes should not be appearing. Are you running black with `black -l 100 file.py`?

Reply:
I just did. Thanks for catching that!